### PR TITLE
Fix unknow header and add info on `register_commands`

### PR DESCRIPTION
**Pull request by Jerry-py**

### CODE REVIEW

File: docs/index.md
Comment (MarzaElise) [inline docs/index.md]: This is to follow convention and link to **all** anchor tags in the file. Revert this changes

File: docs/setups/basicbotsetup.md
Comment (MarzaElise) [inline docs/setups/basicbotsetup.md]: Duplicate line?

File: docs/setups/basicbotsetup.md
Comment (MarzaElise) [inline docs/setups/basicbotsetup.md]: why `ignore` and `stockfish_path` kwargs? im pretty sure it dont matter if we have it or not...

**Review by MarzaElise — CHANGES_REQUESTED** [review]

File: docs/index.md
Comment (MarzaElise) [reply docs/index.md]: wait a sec

Comment (MarzaElise) [reply docs/index.md]: ya ok, just remove the contents part... just have it like 
- [Disgames](#disgames)
    - [Installation](#installation)
    - [Extra](#extra)

Comment (Jerry-py) [reply docs/index.md]: K

File: docs/setups/basicbotsetup.md
Comment (Jerry-py) [reply docs/setups/basicbotsetup.md]: Whoops

File: docs/setups/basicbotsetup.md
Comment (Jerry-py) [reply docs/setups/basicbotsetup.md]: https://github.com/andrewthederp/Disgames/blob/71bb3c5831eee98a0e07819cd263a978c1dc7e2a/disgames/__init__.py#L8

File: docs/index.md
Comment (MarzaElise) [inline docs/index.md]: this should be sh tho, its a shell command

**Review by MarzaElise — APPROVED** [review]

**Comment by MarzaElise** [comment]
everything looks good, just waiting for final review by @andrewthederp  before merding :+1:

File: docs/index.md
Comment (Jerry-py) [reply docs/index.md]: alr will fix but no difference

**Comment by Jerry-py** [comment]
Ok

**Pull request merged**
